tests/cargo-kani/simple-config-toml/Cargo.toml

```diff
@@ -9,13 +9,6 @@ edition = "2018"
 
 [workspace]
 
-[kani.flags.loudness]
-quiet = false
-
-[kani.flags.artifacts.extra.suffixes]
```

Comment:
Just a note for anyone reviewing: these things under `kani.flags` were pointless and I'm not sure why they were ever there. So I removed these "sections" that shouldn't be there.

Also, I removed `--no-quiet` and `--no-visualize` and `mangler` as options, so I removed those here too.